backend/src/entries/entries.controller.ts

```diff
@@ -36,12 +36,22 @@ export class EntriesController {
   async updateEntry(
     @Param('id', ParseIntPipe) id: number,
     @Body() entry: UpdateEntryDto,
-  ): Promise<EntryDto> {
+  ): Promise<Omit<Entry, 'id' | 'checkInTime'>> {
```

Comment:
```suggestion
  ): Promise<Omit<Entry, 'id' | 'checkInTime'>> {
```

Omitを2回しているので、どっかに型としてまとめたいです